lib/common/phpstan.neon.dist

```diff
@@ -8,3 +8,5 @@ parameters:
 		- %currentWorkingDirectory%/src/
 	autoload_files:
 		- %currentWorkingDirectory%/vendor/autoload.php
+	ignoreErrors:
+		- '#^PHPDoc tag @throws with type AmpProject\\Exception\\FailedRemoteRequest is not subtype of Throwable$#'
```

Comment:
phpstan complains about `FailedRemoteRequest` not being a subtype of `Throwable`, but `Throwable` is not available in PHP 5.6 so I'm ignoring it for now.